### PR TITLE
 Analytics for new award flow

### PR DESCRIPTION
**Pull request by DilwoarH**
Overall User story
=============

As a product manager I want to be able to get analytics for the new award flow so that I can track the behaviour of the user.

Resources
========
- https://drive.google.com/drive/u/0/folders/1fJM7CTuRzBEJ7xF_7nj0m4sQgjJQV9R5

See https://trello.com/c/2rthNi7A/71-new-states-for-stage-6-on-search-summary-task-list-page

https://trello.com/c/FMuTKjo4/114-analytics-for-new-award-flow

Ticket: https://trello.com/c/FMuTKjo4/114-analytics-for-new-award-flow

bump to 31.1.0

### CODE REVIEW

File: app/assets/javascripts/analytics/_events.js [hunk 1]
```diff
@@ -89,7 +89,7 @@
 
   GOVUK.GDM.analytics.LinkClick = LinkClick;
 
-  GOVUK.GDM.analytics.events = {
+  GOVUK.GDM.analytics.buyerSpecificEvents = {
```
Comment (galund) [inline app/assets/javascripts/analytics/_events.js]: I think the name here is supposed to correspond to the module/file name, to make understanding/debugging more sane?

Comment (DilwoarH) [reply app/assets/javascripts/analytics/_events.js]: @galund - updated

File: app/assets/javascripts/analytics/_init.js [hunk 2]
```diff
@@ -5,6 +5,7 @@
     this.register();
     this.pageViews.init();
     this.events.init();
```
Comment (galund) [inline app/assets/javascripts/analytics/_init.js]: So does this work / is it initialising the FE toolkit events.js, and if so shouldn't that be happening in `toolkit/javascripts/analytics/_init.js`?

Comment (DilwoarH) [reply app/assets/javascripts/analytics/_init.js]: Hi @galund - yes this is needed as this is an override function. If the init is not done here then the function would not trigger.

Comment (galund) [reply app/assets/javascripts/analytics/_init.js]: Comment here - and perhaps also in the effectively-overridden init function? (in the other repo I guess) - and then we're good. I'll approve now so you don't have to wait for me any more.

Comment (DilwoarH) [reply app/assets/javascripts/analytics/_init.js]: Comment left here

https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/443#issuecomment-398343433

**Pull request merged**